PerfNext/public/lib/js/util.js

```diff
@@ -640,15 +640,18 @@ function insertCapabilityEnvVars(capabilityName, hwEnvClassName, idToFind) {
 	
 	var hostNode = getHostNode() ;
 	
-	//Will match all ids starting with the capabilityName
-	var capabilityNode = hostNode.querySelector('[name^="' + capabilityName + '"]');
+	var capabilities = hostNode.querySelectorAll('capability');
+	
 	var envVARs = 0;
-	if (capabilityNode)
-	{
-		envVARs = capabilityNode.children;
-		console.log("util.js insertCapabilityEnvVars capabilityNode envVARs.length: "+envVARs.length); 
+	for (const capability of capabilities) {
```

Comment:
Could you please add more comments for what we're doing in this loop so that it's easier to others to understand?

Reply:
Just mention that we only care about the benchmark name and not the variant since HW specific vars don't usually change for variants